src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/controller/jobs/MatchingCandidateJobDetailsGetShould.kt

```diff
@@ -14,48 +18,43 @@ class MatchingCandidateJobDetailsGetShould : MatchingCandidateJobDetailsTestCase
 
   @Test
   fun `retrieve details of a matching candidate job`() {
-    val jobId = "04295747-e60d-4e51-9716-e721a63bdd06"
-
     assertGetMatchingCandidateJobDetailsIsOK(
-      id = jobId,
-      parameters = null,
-      expectedResponse = tescoWarehouseHandlerJobDetailsBody(
-        id = jobId,
-        expressionOfInterest = false,
-        archived = false,
-      ),
+      id = tescoWarehouseHandler.id.id,
+      parameters = "prisonNumber=$prisonNumber",
+      expectedResponse = builder()
+        .from(tescoWarehouseHandler)
+//        .withDistance(distanceInMiles)
```

Comment:
do we need this later?

Reply:
This is part of the next PR I'm working on. I prefer to leave it because is nor harming and will be integrated in the following days.